types/vector2js/tsconfig.json

```
@@ -0,0 +1,23 @@
+{
+    "compilerOptions": {
+        "module": "commonjs",
+        "lib": [
+            "es5"
```

```suggestion
            "es6"
```
the native package is in ES5 clearly, but here we're describing the shape of the JS api, so we can use ES6 features, like class, hence the 'es6' is default on DT